psi4/driver/driver_findif.py

```diff
@@ -238,13 +245,72 @@ def _geom_generator(mol, freq_irrep_only, mode):
 
     Returns
     -------
-    disp_geoms : list of psi4.core.Matrix
-        A list of displaced geometries to compute quantities at.
+    findifrec : dict
+        Dictionary of finite difference data, specified below.
+        The dictionary makes findifrec _extensible_. If you need a new field
+        in the record, just add it.
+        All fields should be present at all times, with two exceptions:
+            1. Fields for computed quantities will not be available until
+               after they are computed.
+            2. Displacement specific overrides for globals will not be
+               available unless the user specified the overrides.
+               (Such overrides are not implemented at time of writing. An example
+               is giving a displacement its own step dict.)
+
+    step : dict
+        A descriptor for the finite difference step.
+        In future, this can be overriden by step fields for individual displacements.
+
+        units : string
+            The units for the displacement. The code currently assumes "bohr".
+        size : float
+            The step size for the displacement.
+
+    stencil_size : int
+        Number of points to evaluate at for each displacement basis vector. The count
+        includes the reference. Psi currently supports 3 and 5.
+
+    displacement_space : string
+        A string specifying the vector space in which displacements are performed.
+        Currently, only CdSalc is supported.
+
+    project_translations : bool
+        Whether translations are to be projected out of the displacements.
+
+    project_rotations : bool
+        Whether rotations are to be projected out of the displacements.
+
+    molecule : dict
+        The reference molecule, in MolSSI schema. See
+        https://molssi-qc-schema.readthedocs.io/en/latest/auto_topology.html
+
+    displacements : dict
+        A dictionary mapping labels specifying the displacement to data about
+        the geometry. Labels are of the form "A: B, C: D" where A and B index the
+        basis vector in displacement space and are ordered, and C and D index the
```

Comment:
sure you don't mean A & C index in displacement space and B & D in magnitude? might want to use `A: a, B: b`